### PR TITLE
Reflections

### PR DESCRIPTION
**Pull request by TMGWill**
This pull contains my response for my Week 1 Reflection.

### CODE REVIEW

**Comment by TMGWill** [comment]
Week 2 Reflection is in pull request.

File: Reflections/Week_1_Reflection.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+**Will Pratt**
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+* I have managed to translate my proof of concept to classes I created myself for more of an object oriented programming approach. I have also begun implementing the rooms class from adventurelib within my project.
```
Comment (walkerje) [inline Reflections/Week_1_Reflection.md:3]: Awesome! I've mentioned this before but I'd like to reiterate; putting a graph together on paper (or even in draw.io) would likely be highly beneficial for figuring out how rooms "connect" to each other, as well as room states!

File: Reflections/Week_1_Reflection.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+* The main challenge that I had was trying to create classes that would work for what I was trying to do and trying to decide what the best route would be. I referenced a previous project of mine where I made my own class to help remind myself of what I can do. It also involved a good deal of trial and error to see what seemed right and what didn't.
+3. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
+* I will work on refining the combat mechanics for the player which could take 1-2 days. I will work on debugging the possible inputs the player could put in when moving between different rooms which could take 1-2 days. I will also work on trying to make rooms more specific so I can use them with a more object oriented kind of appraoach which might take 1-3 days. I will also work on commenting my code which will be a continuous task.
```
Comment (walkerje) [inline Reflections/Week_1_Reflection.md:7]: Great goals! Keep chipping away at building an understanding of AdventureLib, you're making great progress in fitting your design ideas into AdventureLib's way of "doing things".

File: Reflections/Week_2_Reflection.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+**Will Pratt**
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+* I have begun adding comments and have gone about creating states for rooms and checking them to determine what should be done depending on the room.
```
Comment (walkerje) [inline Reflections/Week_2_Reflection.md:3]: Great progress on getting states working so far. I would like to point to [this part](https://adventurelib.readthedocs.io/en/stable/rooms.html#storing-attributes-on-rooms) of the AdventureLib documentation for special consideration.

**Review by walkerje — APPROVED** [review]
Great work! I apologize for the long delay between getting these accepted-- you can expect they will be reviewed every Sunday afternoon from hereon out. You have good, quantifiable goals that are attainable within the next week, and I'm looking forward to see your project continue to grow. Be sure to take special care reading the adventurelib documentation, and if you have any questions, know that you can always reach out.

**Pull request merged**